uniffi_bindgen/src/bindings/python/templates/ForeignExecutorTemplate.py

```diff
@@ -1,35 +1,35 @@
 # FFI code for the ForeignExecutor type
 
-{{ self.add_import("asyncio") }}
```

Comment:
I think the `add_import` machinery should continue to be used, but more generally, this doesn't really seem idiomatic to me - I've almost never seen people rename every single import they do to support this use-case. Same as for `ctypes`. If every single module designed to be used by other modules was forced to have *every single* import be of the form `import mod as _mod` I think we'd all go a little crazy.